EliteDangerous/JournalEvents/JournalScan_Display.cs

```diff
@@ -436,7 +436,14 @@ public string MassEMText()
                 information.Append(" (Discovered)".T(EDCTx.JournalScanInfo_DIS));
                 if (showvalues)
                 {
-                    information.Append(' ').Append(ev.EstimatedValueFirstMappedEfficiently.ToString("N0")).Append(" cr");
+                    information.Append(' ').Append((ev.EstimatedValueFirstMappedEfficiently > 0 ? ev.EstimatedValueFirstMappedEfficiently : ev.EstimatedValueBase).ToString("N0")).Append(" cr");
```

Comment:
Why did you add this? is this for pre-3.2 scans?

Reply:
Discovered stars are never mapped, so this would query FME for all discovered stars, showing 0 as result. So it's either this to fall back to the correct base value for discovered stars or change the condition to
js.WasDiscovered == true && js.WasMapped == false && js.IsPlanet
and add a second condition
js.WasDiscovered == true && js.WasMapped == false && js.IsStar